mulighetsrommet-arena-adapter/src/main/kotlin/no/nav/mulighetsrommet/arena/adapter/consumers/TiltakdeltakerEndretConsumer.kt

```diff
@@ -71,7 +71,7 @@ class TiltakdeltakerEndretConsumer(
         val mrDeltaker = deltaker.toDomain(tiltaksgjennomforingMapping.entityId, norskIdent)
 
         val method = if (decoded.operation == ArenaEventData.Operation.Delete) HttpMethod.Delete else HttpMethod.Put
-        client.request(method, "/api/v1/arena/deltaker", mrDeltaker)
+        client.request(method, "/api/v1/internal/arena/deltaker", mrDeltaker)
```

Comment:
Denne feiler nok fordi noen av testene også må oppdateres til å lage mockresponser for ny url